doc/sphinx/changes.rst

```diff
@@ -8,6 +8,687 @@ Recent changes
 
    .. include:: ../unreleased.rst
 
+Version 8.13
+------------
+
+Summary of changes
+~~~~~~~~~~~~~~~~~~
+
+Coq version 8.13 integrates many usability improvements, as well
+as extensions of the core language.
+The main changes include:
+
+  - :ref:`Introduction <813PrimArrays>` of :ref:`primitive persistent arrays<primitive-arrays>`
+    in the core language, implemented using imperative persistent arrays.
+  - Introduction of :ref:`definitional proof irrelevance <813UIP>` for the equality type
+    defined in the SProp sort, using the :flag:`Definitional UIP` flag,
+    (deactivated by default). This models definitional uniqueness of
+    identity proofs for this type. It is deactivated by default as
+    it can lead to non-termination in combination with impredicativity.
+    Use of this flag is also printed by :cmd:`Print Assumptions`.
```

Comment:
It's a long explanation, actually longer than the detailed changelog entry. Shouldn't most of this information be moved down to the detailed changelog?

Reply:
Yes